.github/workflows/build_and_test.yaml

```diff
@@ -12,14 +12,15 @@ jobs:
   # StackStorm-Exchange/ci/.github/workflows/pack-build_and_test.yaml@master
 
   build_and_test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     name: 'Build and Test / Python ${{ matrix.python-version-short }}'
     strategy:
       matrix:
         include:
-          - python-version-short: "3.6"
-            python-version: 3.6.13
-            vault-version: 1.5.9
+          - python-version-short: "3.8"
```

Comment:
Worth mentioning in the Changelog the pack is py 3.8-ready, eg tested with 3.8.

Reply:
Changelog entry added.